packages/squidex/schema/gp2/schemas/users.json

```diff
@@ -202,6 +202,123 @@
           "isHalfWidth": true
         }
       },
+      {
+        "name": "country",
+        "isHidden": false,
+        "isLocked": false,
+        "isDisabled": false,
+        "partitioning": "invariant",
+        "properties": {
+          "fieldType": "String",
+          "isUnique": false,
+          "isEmbeddable": false,
+          "inlineEditable": true,
+          "createEnum": false,
+          "contentType": "Unspecified",
+          "editor": "Input",
+          "label": "Country",
+          "isRequired": true,
+          "isRequiredOnPublish": false,
+          "isHalfWidth": true
+        }
+      },
+      {
+        "name": "city",
```

Comment:
hang on, didnt we say location? Although I would much prefer having it as a city, but if we are going to show it as a Location in FE then maybe we want to have it consistent all across the stack